### DISCUSSION TITLE
Verify cached fetch in Chrome

### DISCUSSION THREAD

**Discussion by alextrastero** [vercel/swr#846] [Q&A]
Is there a way to verify a fetch is cached in Chrome dev tools?

**Comment by shuding** [comment]
Currently if you have React Devtools installed, you can see your SWR cached data from the "hooks" section of your component.

**Reply by alextrastero** [reply]
Thank you for replying, I don't supposed there's a `cached: true` flag? Otherwise I don't see a way to confirm that a call was cached.
